bullet_train-themes-light/app/views/showcase/previews/partials/_notices.html.erb

```diff
@@ -0,0 +1,10 @@
+<% showcase.sample "Basic" do %>
+  <% flash[:notice] = "Success Message" %>
+  <% flash[:error] = "Error message" %>
```

Comment:
Let's use `now` so the flash messages don't linger onto the next page visit.

```suggestion
  <% flash.now[:notice] = "Success Message" %>
  <% flash.now[:error] = "Error message" %>
```

Reply:
By the way, it could be interesting to add another sample to `previews/partials/_page.html.erb` ala `sample "With shared/notices"` that also sets these ^ `flash.now` calls, to show what that looks like.